tests/bindings/lua/test_upb.lua

```diff
@@ -661,15 +664,17 @@ function test_msg_primitives()
 end
 
 function test_msg_array()
-  local msgdef = upb.build_defs{
+  local symtab = upb.SymbolTable{
     upb.MessageDef{full_name = "TestMessage", fields = {
       upb.FieldDef{name = "i32_array", number = 1, type = upb.TYPE_INT32,
                    label = upb.LABEL_REPEATED},
       }
     }
   }
 
-  msg = upb.Message(msgdef)
+  factory = upb.MessageFactory(symtab)
+  TestMessage = factory:get_message_class(symtab:lookup("TestMessage"))
+  msg = TestMessage()
```

Comment:
Good point. I changed this one to take a name (only). Can extend it later to take a msgdef also if desired (can dispatch based on its type).